tests/unittest/_torch/thop/serial/test_sparse_kv_cache_compact.py

```
@@ -0,0 +1,442 @@
+        pools_cpu,
+        page_tables_cpu,
+        source_indices,
+        source_offsets,
+        destination_base,
+    )
+    arguments = _device_arguments(pools, source_indices, source_offsets)
+
+    _compact(pools, page_tables, arguments, destination_base)
+    torch.cuda.synchronize()
+
+    for actual, reference in zip(pools, expected):
```

Ruff (0.15.21)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)